i18n/events/overview.i18n.js

```diff
@@ -36,36 +35,33 @@ export default genI18nMessages({
             keynote: {
                 title: 'Keynotes',
                 description:
-                    'PyCon APAC 2022 invites four speakers to give keynote speeches during ' +
-                    'the two-day conference. Each keynote speaker is considered one of ' +
-                    'the most important figures in their respective fields. ' +
-                    'They will share their professional experience and the image of their domain’s future.',
+                    'The two-day agenda featured four speakers giving keynote speeches at different sessions. ' +
+                    'The speakers are all top-notch people in various fields around the world; ' +
+                    'they will share their experience in their fields, projects, or the direction of future development.',
             },
             talk: {
                 title: 'Talks',
                 description:
-                    'The two conference days are packed with talks about Python ' +
-                    'by speakers from Taiwan and around the world. ' +
+                    'Lectures are the most important activities during the two-day session. ' +
                     'The talks will be either 15-, 30-, or 45-minute long. ' +
                     'Three tracks of talks will be delivered simultaneously, ' +
-                    'all with different topics and difficulties.',
+                    'You can choose the topics from different difficulties and types of speech you are interested in listening to.',
             },
             tutorial: {
                 title: 'Tutorials',
                 description:
-                    'Professional courses need to be registered in advance to ' +
-                    'obtain the qualifications. ' +
-                    'The course is usually provided for 1.5 hours. ' +
+                    'The utorial is usually provided for 1.5 hours. ' +
                     'The difference between it and talk is that the speaker will ' +
                     'bring the audience to practice together, ' +
-                    'and they can communicate with each other while learning.',
+                    'and they can communicate with each other while learning. ' +
+                    'Tutorials need to be registered in advance to ' +
+                    'obtain the qualifications. ',
```

Comment:
Is this statement true? I do not see any entry for the tutorials on the [pycontw kktix](https://pycontw.kktix.cc/).

Reply:
Seems like this is last year version. Let me update that.